app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/views/SearchSoftwareView.scala.html

```diff
@@ -24,24 +24,18 @@
 @import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.views.html.templates._
 @import views.html.helper.CSPNonce
 @import uk.gov.hmrc.play.language.LanguageUtils
-@import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.config.featureswitch.FeatureSwitch.IntentFeature
-@import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.config.featureswitch.FeatureSwitchingImpl
 
 @this(
   mainTemplate: MainTemplate,
-  softwareVendorsTemplate: SoftwareVendorsTemplate,
   softwareFiltersTemplate: SoftwareFiltersTemplate,
-  softwareVendorsZeroResultsTemplate: SoftwareVendorsZeroResultsTemplate,
   softwareVendorsIntentResultsTemplate: SoftwareVendorsIntentResultsTemplate,
   softwareVendorsAgentTemplate: SoftwareVendorsAgentTemplate,
-  softwareTabsTemplate: SoftwareTabsTemplate,
   form: FormWithCSRF,
   govukCheckboxes: GovukCheckboxes,
   govukAccordion: GovukAccordion,
```

Comment:
Can remove govukAccordian and govukDetails

Reply:
done